@here/harp-mapview/lib/MapView.ts

```diff
@@ -2118,6 +2119,17 @@ export class MapView extends EventDispatcher {
 
         dataSource.attach(this);
         dataSource.setEnableElevationOverlay(this.m_elevationProvider !== undefined);
+        if (
+            dataSource.addGroundPlane === true &&
+            this.m_tileDataSources.find(
+                ds => ds.addGroundPlane === true && !(ds instanceof BackgroundDataSource)
+            ) !== undefined
+        ) {
+            // eslint-disable-next-line no-console
+            console.warn(
```

Comment:
maybe it would help to report directly the names of the two conflicting datasources in the log?